examples/channels-correlation/channels_correlation.bal

```diff
@@ -0,0 +1,37 @@
+import ballerina/http;
+
+//Defines a channel with json constrained type.
+channel<json> jsonChannel;
+
+service<http:Service> channelService bind { port: 9090 } {
+
+    receive (endpoint caller, http:Request request) {
+
+           //A key can be associated with a channel action, this is used to correlate receivers and senders.
```

Comment:
Shall we break the sentence as follows:
"//A key can be associated with a channel action.
 //You can use a key to correlate receivers and senders."